docs/packages/dara-components/graphs/introduction.mdx

```diff
@@ -126,6 +126,99 @@ def graph_with_meta() -> CausalGraph:
     return cg
 ```
 
+### Graph Legend
+
+By default the graph has a legend which is dependent on the current `editor_mode` selected for that graph. You can gain control over these in two ways:
+
+- You can add to the legends already present with the `additional_legends` property. This accepts an array of legends as can be seen in the example below.
+- Alternatively you can alter the prepopulated legends by passing `default_legends`. This accepts a dict of editor_mode to array of legends.
+
+#### Legend types
+
+There are three kinds of legends which can be added to the graph. Below you will find each kind and the properties they possess.
+
+##### `Legend.Spacer`
+
+Spacers can be used to create a space in between sections of your legend. For example to separate node related legends to edge related ones.
+What it means is that it will not display a symbol. You may still write a label for it.
+
+- `label: string` - Optional label to show in the legend
+
+##### `Legend.Node`
+
+Displays a node symbol, this can be useful to explain what different colors of nodes might mean to an user.
+
+- `label: string` - Optional label to show in the legend
+- `color: string` - Optional color to fill the node symbol in the legend, defaults to theme.blue4
+- `highlight_color: string` - Optional color for the node symbol rim in the legend, defaults to theme.primary
+
+:::tip
+
+When configuring the color properties for legends, the following are supported:
+
+**Standard Colors** - You can use common color formats such as named colors ('red', 'blue', etc.),
+hexadecimal values ('#ff0000', '#0000ff'), RGB ('rgb(255, 0, 0)'), or any other CSS-compatible color specification.
+This is ideal for specific, custom color requirements.
+
+**Themed Colors** - These are specified with a `theme.` prefix, followed by the color key from our default theme palette.
```

Comment:
Technically it doesn't necessarily use the default theme; it will use whatever theme is configured for the app, could be a custom one. It will just look up that particular color in the theme.

Reply:
Added a little section expanding on these points, let me know what you think